model/riscv_vreg_type.sail

```diff
@@ -0,0 +1,160 @@
+type vreglen : Int = vlenmax /* use the largest possible register length */
+type vreglenbits = bits(vreglen)
+
+/* default vector register type */
+type vregtype = vreglenbits
+
+/* default vector register printer */
+val VRegStr : vregtype -> string
+function VRegStr(r) = BitStr(r)
+
+/* floating point types */
+
+enum FPType = {
+  FPNONZERO,
+  FPZERO,
+  FPINF,
+  FPQNAN,
+  FPSNAN
+}
+
+enum VDATATYPE    = { FP_VDATATYPE }
+
+val get_vector_datatype : unit -> VDATATYPE
```

Comment:
Does this need to be a function? @XinlaiWan

Reply:
This unnecessary function is deleted.